addons/test_discuss_full/tests/test_performance.py

```diff
@@ -964,18 +964,7 @@ def _get_init_messaging_result(self):
                 },
             ],
             'companyName': 'YourCompany',
-            'shortcodes': [
-                {
-                    'id': 1,
-                    'source': 'hello',
-                    'substitution': 'Hello, how may I help you?',
-                },
-                {
-                    'id': 2,
-                    'source': 'bye',
-                    'substitution': 'Thanks for your feedback. Goodbye!',
-                },
-            ],
+            'shortcodes': [],
```

Comment:
It would be nice to have at least one returned in the test to see how it looks like and that it still works (and how many query it takes to display them).